README.md

```diff
@@ -2,24 +2,249 @@
+7. The `paths-ignore` are the path list that will be ignored by license-eye. By default, `.git` and the content in `.gitignore` will be inflated into the `paths-ignore` list.
+8. On what condition License-Eye will comment the check results on the pull request, `on-failure`, `always` or `never`. Options other than `never` require the environment variable `GITHUB_TOKEN` to be set.
+9. `dependency` section is configurations for resolving dependencies' licenses.
+10. `files` are the files that declare the dependencies of a project, typically, `go.mo` in Go project, `pom.xml` in maven project, and `package.json` in NodeJS project. If it's a relative path, it's relative to the `.licenserc.yaml`.
+
+**NOTE**: When the `SPDX-ID` is Apache-2.0 and the owner is Apache Software foundation, the content would be [a dedicated license](https://www.apache.org/legal/src-headers.html#headers) specified by the ASF, otherwise, the license would be [the standard one](https://www.apache.org/foundation/license-faq.html#Apply-My-Software).
+
+## Supported File Types
+
+The `header check` command theoretically supports all kinds of file types, while the supported file types of `header fix` command can be found [in this YAML file](assets/languages.yaml). In the YAML file, if the language has a non-empty property `comment_style_id`, and the comment style id is declared in [the comment styles file](assets/styles.yaml), then the language is supported by `fix` command.
+
+- [assets/languages.yaml](assets/languages.yaml)
+
+  ```yaml
+  Java:
+    type: programming
+    tm_scope: source.java
+    ace_mode: java
+    codemirror_mode: clike
+    codemirror_mime_type: text/x-java
+    color: "#b07219"
+    extensions:
+      - ".java"
+    language_id: 181
+    comment_style_id: SlashAsterisk
+  ```
+
+- [assets/languages.yaml](assets/languages.yaml)
+
+  ```yaml
+  - id: SlashAsterisk     # (i)
+    start: '/*'           # (ii)
+    middle: ' *'          # (iii)
+    end: ' */'            # (iv)
+  ```
+
+  1. The `comment_style_id` used in [assets/languages.yaml](assets/languages.yaml).
+  2. The leading characters of the starting of a block comment.
+  3. The leading characters of the middle lines of a block comment.
+  4. The leading characters of the ending line of a block comment.
+
+## Contribution
+
+- If you find any file type should be supported by the aforementioned configurations, but it's not listed there, feel free to [open a pull request](https://github.com/apache/skywalking-eyes/pulls) to add the configuration into the two files.
+- If you find the license template of an SPDX ID is not supported, feel free to [open a pull request](https://github.com/apache/skywalking-eyes/pulls) to add it into [the template folder](assets/header-templates).
+
+## License
+
+[Apache License 2.0](https://github.com/apache/skywalking/blob/master/LICENSE)
```

Comment:
This url should link to the license file in this repo, right?

Reply:
Yes good catch, fixed